types/react-twitter-embed/index.d.ts

```diff
@@ -0,0 +1,85 @@
+// Type definitions for react-twitter-embed 3.0
+// Project: https://github.com/saurabhnemade/react-twitter-embed#readme
+// Definitions by: Cyrille <https://github.com/zozoens31>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.5
```

Comment:
Is 3.5 really the minimum version of TypeScript required to compile these types? That seems overly restrictive.